Files with single cell data for gbm_cptac_2021 study

jessicaTH opened the pull request:
# What?
Fix #1595 

Cancer studies updated in this pull request:
- gbm_cptac_2021 

# checks
For all pull requests:
- [x] Passes validation

The team at MSK and The Hyve are working on developing a feature where a custom tab is enabled in the Patient view page that shows single-cell data. This gbm_cptac_2021 study has single-cell data available for 18 patients. The last step to making this production-ready is adding some generic assay data to link the cBioPortal study to the corresponding single-cell data. (https://github.com/cBioPortal/cbioportal/issues/9246)

The files I have added are:
gbm_cptac_2021/case_lists/cases_single_cell_data.txt
gbm_cptac_2021/data_single_cell_phase.txt
gbm_cptac_2021/data_single_cell_type.txt
gbm_cptac_2021/meta_single_cell_cycle_phases.txt
gbm_cptac_2021/meta_single_cell_type_frac.txt

yichaoS commented:
@egarcialara The profile didn't show up after importing to triage. Need to fix that that -- I'm trying to figure out maybe it is because of file names.

yichaoS commented:
@jessicaTH @egarcialara I've updated the configuration sheet on our side and normalized the names. Could you update the names to:
`data_single_cell_cycle_phases.txt``data_single_cell_type_fractions.txt``meta_single_cell_cycle_phases.txt``meta_single_cell_type_fractions.txt` and the profile `stable_id` in the meta files into `single_cell_type_fractions` and `single_cell_cycle_phases`.

jessicaTH commented:
@yichaoS thanks for updating the configuration, I have changed the file names and updated the details in the metafiles. Let me know if there's anything else I could do!

yichaoS commented:
@jessicaTH The study is imported to the testing instance and it looks good overall. 
There are two small things. 
In `data_single_cell_type_fractions.txt`, could you remove the NAME, DESCRIPTION and URL column (also in the corresponding meta file), and change the IDs text in the header columns to camel cases and remove the underscores, for example, `Smooth_muscle_cells` to `Smooth Muscle Cells`.

jessicaTH commented:
@yichaoS - Done! 
Is the testing environment (https://beta.cbioportal.org/)? I wanted to see the files. Let me know if there's anything else I can fix

yichaoS commented:
@jessicaTH It is the internal testing portal instance we are using. I attached some screenshots here of what it looks like. Sorry I didn't get what you mean why "seeing the files"? The data files behind the visualization gonna be these ones in the PR, once merged.
